### PR TITLE
Fix issue of not displaying/updating enterprise user complex objects in myaccount and console

### PR DESCRIPTION
**Pull request by AnuradhaSK**
## Purpose
Fixes https://github.com/wso2/product-is/issues/11266

### CODE REVIEW

File: apps/myaccount/src/components/profile/profile.tsx [hunk 1]
```diff
@@ -153,13 +153,20 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): J
...
-                            && profileDetails.profileInfo[schemaNames[0]]
-                                .find((subAttribute) => subAttribute.type === schemaNames[1]);
-                        tempProfileInfo.set(
-                            schema.name,
-                            subValue ? subValue.value : ""
-                        );
+                        if (schema.extended) {
+                            tempProfileInfo.set(schema.name,
+                                profileDetails?.profileInfo[ProfileConstants.SCIM2_ENT_USER_SCHEMA]?.[schemaNames[0]]
+                                    ? profileDetails?.profileInfo[ProfileConstants.SCIM2_ENT_USER_SCHEMA][schemaNames[0]][schemaNames[1]]
```
Comment (brionmario) [inline apps/myaccount/src/components/profile/profile.tsx:159]: Possible max line length issue here.

File: apps/console/src/features/users/components/user-profile.tsx [hunk 2]
```diff
@@ -229,13 +229,22 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
...
-                        tempProfileInfo.set(
-                            schema.name,
-                            subValue ? subValue.value : ""
-                        );
+                        if (schema.extended && userInfo[ProfileConstants.SCIM2_ENT_USER_SCHEMA]) {
+                            const complexEnterprise = schemaNames[0] && schemaNames[1] &&
+                            userInfo[ProfileConstants.SCIM2_ENT_USER_SCHEMA][schemaNames[0]] &&
+                                userInfo[ProfileConstants.SCIM2_ENT_USER_SCHEMA][schemaNames[0]][schemaNames[1]] &&
+                            (
+                                tempProfileInfo.set(schema.name, userInfo[ProfileConstants.SCIM2_ENT_USER_SCHEMA][schemaNames[0]][schemaNames[1]])
```
Comment (brionmario) [inline apps/console/src/features/users/components/user-profile.tsx:237]: Possible max line length issue here.

File: apps/myaccount/src/components/profile/profile.tsx (hunk 1 above)
Comment (AnuradhaSK) [reply apps/myaccount/src/components/profile/profile.tsx]: fixed in a1776ad

File: apps/console/src/features/users/components/user-profile.tsx (hunk 2 above)
Comment (AnuradhaSK) [reply apps/console/src/features/users/components/user-profile.tsx]: fixed in a1776ad

**Review by jeradrutnam — APPROVED** [review]

**Pull request merged**